assets/section-main-product.css

```diff
@@ -378,6 +378,10 @@ a.product__text {
   text-align: center;
 }
 
+.product--no-media shopify-payment-terms {
+  text-align: center;
+}
```

Comment:
Looks good to me but I think we could add this selector in the declaration above so they're grouped 🙂